sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1571,6 +1616,89 @@ a normal SYCL command-group submission.
 associated with the graph node resulting from this command-group submission is
 different from the one with which the dynamic_parameter was created.
 
+|===
+
+=== Dynamic Events [[dynamic-events]]
+
+[source,c++]
+----
+namespace ext::oneapi::experimental {
+  class dynamic_event {
+    dynamic_event();
+    dynamic_event(const event& ev);
+
+    void update(const event& ev);
+  };
+}
+----
+
+Dynamic events represent SYCL events from outside of a given `command_graph`
+(either obtained from normal SYCL operations or from another `command_graph`)
+that nodes in that graph may depend on. The `dynamic_event` object enables these
+dependent events to be updated between graph executions.
+
+Dynamic events can be used to add dependencies to a graph node in the same way
+that regular SYCL events can, by passing them as parameters to
+`handler::depends_on()` inside the CGF which represents the node.
+
```

Comment:
For dynamic parameters we have the statement:

> The dynamic_parameter class provides the [common reference semantics](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:reference-semantics).

Should we have something similar here.

Reply:
Done!